### PR TITLE
playlist countdown screen displayed for 1-2 sec after the player loaded

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

do not render the component when no duration exists (while loading)

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/playlist-countdown/playlist-countdown.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class PlaylistCountdown extends Component {
    * @returns {boolean} shouldComponentUpdate
    */
   shouldComponentUpdate(nextProps: Object): boolean {
-    return !(nextProps.isSeeking || this.props.isPlaybackEnded);
+    return !!(this.props.duration && !nextProps.isSeeking && !this.props.isPlaybackEnded);
```
Comment (RoyBregman) [inline src/components/playlist-countdown/playlist-countdown.js]: i never understood the purpose of doing a "!!" isn't it just clear without it?

Comment (yairans) [reply src/components/playlist-countdown/playlist-countdown.js]: @RoyBregman This is to make `0` to `false` (`shouldComponentUpdate` expects to `boolean`)

Comment (OrenMe) [reply src/components/playlist-countdown/playlist-countdown.js]: Agree that it's more clear to do this as `this.props.duration > 0`

Comment (yairans) [reply src/components/playlist-countdown/playlist-countdown.js]: done

**Comment by yairans** [comment]
reverted since the title is missing the fec number

**Pull request merged**